extensions/incubator/src/main/java/io/opentelemetry/extension/tracing/ExtendedTracer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.extension.tracing;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanBuilder;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Scope;
+import java.util.concurrent.Callable;
+
+/** Provides easy mechanisms for wrapping standard Java constructs with an OpenTelemetry Span. */
+public final class ExtendedTracer implements Tracer {
+
+  private final Tracer delegate;
+
+  /** Create a new {@link ExtendedTracer} that wraps the provided Tracer. */
+  public static ExtendedTracer create(Tracer delegate) {
```

Comment:
personal taste :) 

s/delegate/base?

Reply:
hm. I personally prefer `delegate` since this is the GoF delegate pattern, but I don't feel overly strongly about it.

Reply:
Leave it delegate, for me was more that this is the `base` tracer and extended :) leave it as delegate.